src/stylesheets/main.css

```diff
@@ -2128,3 +2254,6 @@ input.date-range-input-text {
 
 .font-small {
   font-size: 0.85rem; }
+
+.text-center {
+  text-align: center; }
```

Comment:
What am I missing about how these get used?

I see this, and other classes newly defined, but I don't see them consumed anywhere in the PR.

Reply:
Good catch. This must be orphaned css from some other idea. It's not used anywhere in the codebase. I'll remove it.